x-pack/metricbeat/module/aws/billing/_meta/docs.asciidoc

```diff
@@ -50,7 +50,7 @@ group by type dimension and type tag with separate config parameters:
 
 * *group_by_dimension_keys*: A list of keys used in Cost Explorer to group by
 dimensions. Valid values are AZ, INSTANCE_TYPE, LEGAL_ENTITY_NAME,
-LINKED_ACCOUNT, OPERATION, PLATFORM, PURCHASE_TYPE, SERVICE, TAGS, TENANCY, and
+LINKED_ACCOUNT, OPERATION, PLATFORM, PURCHASE_TYPE, SERVICE, TENANCY, and
```

Comment:
Thanks @jguay for fixing this. While you are here, could you actually list all the supported dimension keys here please? I think I missed a bunch in this list comparing with https://github.com/elastic/beats/blob/master/x-pack/metricbeat/module/aws/billing/billing.go#L35

Reply:
I added a commit for this now, list in doc should now match